apps/hash-graph/Makefile.toml

```diff
@@ -123,29 +124,25 @@ args = ["llvm-cov", "--no-report", "nextest", "--workspace", "--all-features", "
 extend = "coverage-task"
 args = ["llvm-cov", "report", "${@}"]
 
+
+## OpenAPI client
+####################
+
 # If this step fails, the OpenAPI spec is invalid.
 # When running in CI, a non-empty git diff will fail the pipeline.
 [tasks.generate-openapi-client]
```

Comment:
Just for my understanding, is the intention that we run this task only when the Graph API is running? The script to start/stop the API is now gone.

I see that it's part of the `tasks.test-docker` step, so I suppose this is the intention

Reply:
Yes, this is the intention so we can easily create it either from a "native" graph or from the graph started through docker